Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -196,7 +196,7 @@ class PaymentSheetStandardUITests: PaymentSheetUITestCase {
         // `mc_load_succeeded` event `selected_lpm` should be "apple_pay", the default payment method.
         XCTAssertEqual(analyticsLog[2][string: "selected_lpm"], "apple_pay")
         app.buttons["+ Add"].waitForExistenceAndTap()
-        XCTAssertTrue(app.staticTexts["Add a card"].waitForExistence(timeout: 2))
+        XCTAssertTrue(app.staticTexts["Card information"].waitForExistence(timeout: 2))
```

Comment:
I had to update this UI test. With Panther enabled there's a second payment method available in this scenario now, and so the `Add a card` text was no longer shown. Here's the changes to this screen with Panther:

| Before (without Panther) | Now (with Panther) |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 - 2024-09-25 at 11 28 33](https://github.com/user-attachments/assets/3ced04e1-7127-4d8c-9d9f-dbde1f3422ea) | ![Simulator Screenshot - iPhone 15 - 2024-09-25 at 11 28 03](https://github.com/user-attachments/assets/6a1e3671-ab0e-4c63-bf61-3117a8186eeb) |